db.go

```diff
@@ -40,7 +40,7 @@ func (p Postgres) MigrationLogDeleteSql() string {
 type Mysql struct{}
 
 func (m Mysql) SelectMigrationTableSql() string {
-	return "SELECT table_name FROM information_schema.tables WHERE table_name = ?"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name = ? AND table_schema = ?"
```

Comment:
It seems like `table_schema` refers to the database in MySql (https://dev.mysql.com/doc/refman/5.1/en/information-schema-optimization.html). If that's so, isn't this redundant because the connection input already refers to a database?